crates/crates_io_database/src/schema.rs

```diff
@@ -408,6 +408,8 @@ diesel::table! {
         crate_id -> Int4,
         /// Reference to the version in the `versions` table.
         version_id -> Int4,
+        /// The total number of versions.
+        total -> Int4,
```

Comment:
I think I'd prefer something like `num_versions` since it's a bit more self-descriptive

Reply:
fair enough! I'll update it!